### ISSUE TITLE
[Cause identified - Outdated OpenAL library in the game files] Sound Effects playing in wrong speakers on 5.1 analog sound system

### ISSUE THREAD

**Issue by lukaself** [UnderscoreKilburn/repentance-issues#22]
1.7.5 Audio issues - Sound effects playing in the wrong speakers
Starting with this patch, all sound effects are now played in the back speakers on a Logitech 5.1 sound system. Music plays fine in the front speakers. All other games still work fine. 

Expected behaviour: Sound effects should play in the same channel as music. (Front Left and Front Right)

Not the first time it happened right after a BOI:R patch (or BOI for that matter), hopefully can be fixed once more.

**Comment by lukaself** [comment]
**Issue located:** outdated OpenAL32.dll in the game folder (Extracting the .dll shows all files are dating back from 2014)

**Workaround :** Removing OpenAL32.dll from the game folder forces the game to use the OpenAL32.dll located in Windows\System 32 which is up-to-date (2021). Sound effects are output in the front left and right channels as expected after that. 

**Proposed fix:** Developer should update the game's OpenAL32.dll to the latest version (or the version pre-1.7.5 as it also worked fine and was more recent than the current one)

**Comment by lukaself** [comment]
Fixed in 1.7.7 Thanks and Please don't touch anything xD

**Comment by lukaself** [comment]
Back in 1.7.7a Sound effects are in the back speakers again. TT

**Comment by D-Kowal** [comment]
Still ongoing in 2025, sound effects are coming from the 2 back speakers. I'm using Logitech 5.1 Surround Speaker Set.

//edit I would also be thankful if anyone knows a workaround, please post it here or DM me :)

**Comment by D-Kowal** [comment]
Ok, found a workaround for myself:

Install the OpenAL libraries from if you don't have it: [http://www.openal.org/downloads/oalinst.zip]

I replaced the OpenAL32.dll file in the game folder with a copy of the one that got installed into system32 (while also keeping the original in system32). Probably also would have worked when if I just deleted the game folder DLL after installing the OpenAL librery for Windows.

Basically this workaround is for those who somehow didn't have OpenAL32.dll installed on their PC, apparently that happen :)
